tests/test_suite/run_test_command.py

```diff
@@ -594,9 +559,7 @@ def nncf_validate_fq_testing(template, root, otx_dir, task_type, test_name):
     _validate_fq_in_xml(xml_path, path_to_ref_data, "nncf", test_name)
 
 
-def nncf_eval_testing(
-    template, root, otx_dir, args, threshold=0.001, criteria=None, reg_threshold=0.10, result_dict=None
-):
+def nncf_eval_testing(template, root, otx_dir, args, threshold=0.01):
```

Comment:
Previous `threshold` is 0.001, but it's changed to 0.01. Is it intentional?

Reply:
Yes, our threshold for NNCF is 0.01. previous values were wrong. I'll also edit the E2E, integ tests in this PR(https://github.com/openvinotoolkit/training_extensions/pull/1988)